templates/takeovers/_template.html

```diff
@@ -1,68 +1,75 @@
 <section {% if timezone %}data-timezone="{{ timezone }}"{% endif %} {% if lang %}lang="{{ lang }}"{% endif %} data-lang="{% if locale %}{{ locale }}{% else %}en_GB{% endif %}" class="{% if class %}{{ class }}{% else %}p-takeover--grad{% endif %} js-takeover">
   <div class="row u-equal-height">
     <div class="{% if equal_cols %}col-6{% else %}col-7{% endif %} u-vertically-center">
-      <div>
-        <h1>
-          {{ title | safe }}
-        </h1>
-        {% if subtitle %}<h4>
-          {{ subtitle | safe }}
-        </h4>{% endif %}
-        <div class="u-hide--small">
-          {% if primary_url %}
+      {% if stacked_image %}
+        <div class="u-hide u-show--small">
+          <img class="p-heading-icon__img " src="{{ header_image }}" alt="">
```

Comment:
We should use the full markup for the heading icon pattern. Including the heading. By using this class outside of this block pattern leads to untested and unexpected styling.

Reply:
Made a change. Let me know if this is not what you meant. :D